toxcore/query.h

```diff
@@ -0,0 +1,57 @@
+/** query.h
+ *
+ * Makes requests for information using the DHT or Onion as appropriate.
+ */
+
+#ifndef TOX_QUERY_H
+#define TOX_QUERY_H
+
+#include "network.h"
+#include "tox.h"
+
+#define QUERY_TIMEOUT 500
+
+typedef struct {
+    IP_Port  ipp;
+    uint8_t  key[TOX_PUBLIC_KEY_SIZE];
+    uint8_t  name[TOX_QUERY_MAX_NAME_SIZE];
+    size_t  length;
+
+    uint64_t query_nonce;
+
+    uint8_t  tries_remaining;
+    uint64_t next_timeout;
+} P_QUERY;
```

Comment:
Violation of naming conventions (do we even have those?). We rarely (or never?) have `ALL_CAPS` type names. These are not macro names. Also, what does P stand for? If it stands for PENDING, then why is it abbreviated here but not below?